gnocchi/rest/aggregates/operations.py

```diff
@@ -32,39 +36,207 @@
 }
 
 
-def handle_aggregate(agg, granularity, timestamps, values, is_aggregated):
+# TODO(sileht): expose all operators in capability API
+binary_operators = {
+    u"=": numpy.equal,
+    u"==": numpy.equal,
+    u"eq": numpy.equal,
+
+    u"<": numpy.less,
+    u"lt": numpy.less,
+
+    u">": numpy.greater,
+    u"gt": numpy.greater,
+
+    u"<=": numpy.less_equal,
+    u"≤": numpy.less_equal,
+    u"le": numpy.less_equal,
+
+    u">=": numpy.greater_equal,
+    u"≥": numpy.greater_equal,
+    u"ge": numpy.greater_equal,
+
+    u"!=": numpy.not_equal,
+    u"≠": numpy.not_equal,
+    u"ne": numpy.not_equal,
+
+    u"%": numpy.mod,
+    u"mod": numpy.mod,
+
+    u"+": numpy.add,
+    u"add": numpy.add,
+
+    u"-": numpy.subtract,
+    u"sub": numpy.subtract,
+
+    u"*": numpy.multiply,
+    u"×": numpy.multiply,
+    u"mul": numpy.multiply,
+
+    u"/": numpy.true_divide,
+    u"÷": numpy.true_divide,
+    u"div": numpy.true_divide,
+
+    u"**": numpy.power,
+    u"^": numpy.power,
+    u"pow": numpy.power,
+
+}
+
+# TODO(sileht): around it take a decimal arg
```

Comment:
I don't understand this sentence :)

Reply:
around = numpy.around, it's just another operator I want to add.